cloud_testing/xctest_parser/src/test/kotlin/xctest/ParseTest.kt

```diff
@@ -47,6 +51,45 @@ class ParseTest {
             "EarlGreyExampleSwiftTests/testWithInRoot"
     )
 
+    init {
+        val assetLink = remoteAssetLink()
+        if (updateRequired(assetLink)) {
+            updateFixtures(assetLink)
+        }
+    }
+
+    private fun remoteAssetLink(): Element {
+        val doc = Jsoup.connect("https://github.com/Flank/test_artifacts/releases/tag/v0.1").get()
```

Comment:
I think `https://github.com/Flank/test_artifacts/releases/latest` is the right URL to use, then we're not tied to an individual version.

Reply:
jsoup connect would benefit from retries/backoff. If this network request fails, then the test execution breaks.

https://stackoverflow.com/a/22890355

Reply:
So true. I’ll look at adding that as well.